package.json

```diff
@@ -6,7 +6,12 @@
   "version": "0.8.0",
   "scripts": {
     "start": "react-native start",
-    "lint": "eslint ."
+    "lint": "eslint .",
+    "compile:test-ios": "PLATFORM=ios babel-node cli/compile.js --presets react-native",
```

Comment:
For which language features do we need `babel-node`? Is Node 6.x's ES6 support not sufficient? Just trying to introduce fewer moving parts if we can get away with it.